File: apac/20160918/template.cpp
#include <stdio.h>
#include <string.h>
#include <string>
#include <unordered_map>
#include <map>
#include <iostream>
#include <algorithm>
#include <vector>
#include <iomanip>

using namespace std;

void solve(int kase){
	long long res, L, R;
	scanf("%lld %lld", &L, &R);
	res = min(L, R);
	res = res * (res + 1) / 2;
	printf("Case #%d: %lld\n", kase, res);
}

int main(int argc, char *argv[]){
	if(argc >= 2) {
		freopen(argv[1], "r", stdin);
	}else{
		freopen("A.in", "r", stdin);	
	}
	int kase;
	scanf("%d\n", &kase);
	for(int i=1; i<=kase; i+=1){
		solve(i);	
	}
	return 0;
}

File: apac/20160918/template.py
# coding=utf8

import sys
import numpy as np

def solve(kase):
    n, = scanf("%d")
    a = map(int, raw_input().split())
    a[0] = -a[0]
    print "Case #%d: %.12f" % (kase, np.irr(a))

if __name__ == '__main__':
    if len(sys.argv) >= 2:
        sys.stdin = open(sys.argv[1], 'r')
    if len(sys.argv) >= 3:
        sys.stdout = open(sys.argv[2], 'w')
    
    from scanf import *

    kase, = scanf("%d\n")
    for i in xrange(1, kase+1):
        solve(i)